packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/sections/PreviewDetailsSection.tsx

```diff
@@ -31,9 +32,12 @@ export const PreviewDetailsSection = ({
           },
           {
             label: t(i18n)`Fulfillment date`,
-            value:
```

Comment:
Why did you remove the counterpart info? There is no Jira task, so it's not obvious.

Reply:
The check wasn't correct. I think I just copy-pasted it for fulfillment date so the check for counterparts wasn't needed here